themes/default/content/what-is/what-is-serverless.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "What is Serverless?"
+meta_desc: |
+    Understand serverless architectures, along with some of the benefits of using serverless architectures for modern application development
+type: what-is
+page_title: "What is Serverless?"
+---
+
+Serverless architectures have gained and continue to gain significant traction in modern application development. Despite its name, serverless doesn't mean servers are absent; instead, it represents a paradigm shift in how applications are built, deployed, and scaled in the cloud.
+
+## What is serverless?
+
+_Serverless_ refers to a cloud computing execution model where cloud providers manage the infrastructure automatically, abstracting away the complexity of server management from developers. In a serverless architecture, developers focus solely on writing code for individual functions or services without concerning themselves with server provisioning, scaling, or maintenance.
+
+## How does serverless relate to functions-as-a-service (FaaS)?
+
+The term _functions as a service_ (typically abbreviated as FaaS) focuses specifically on the serverless execution of functions in response to events. FaaS is a subset of serverless, emphasizing the granular, event-driven nature of the architecture. _Serverless_, on the other hand, encompasses the entire architecture, including FaaS. It emphasizes the abstracted, managed nature of the infrastructure.
+
+Some FaaS platforms used in serverless architectures include:
+
+* [AWS Lambda](https://aws.amazon.com/lambda/)
+* [Azure Functions](https://azure.microsoft.com/en-us/products/functions/)
+* [Google Cloud Functions](https://cloud.google.com/functions/)
+* [Netlify Functions](https://functions.netlify.com/)
+* [Cloudflare Workers](https://workers.cloudflare.com/)
+
+## What is event-driven architecture, and is it different than serverless?
+
+_Event-driven architecture_ describes the broader architectural pattern where functions respond to events. Serverless architectures are event-driven architectures. While often associated with serverless, event-driven architectures can exist independently, involving both serverless and traditional infrastructure.
```

Comment:
Perhaps this should be organized into H2 "How is Serverless Different Than..."

And the H3 is "Functions as a service" and "Event-driven architectures"

Reply:
This seems to be adding hierarchy only for the sake of hierarchy. Does it significantly improve the readability of the page, or improve the SEO value of the page?